.packit.yaml

```diff
@@ -42,7 +42,9 @@ jobs:
       job: tests
       trigger: pull_request
       targets:
-        - epel-8-x86_64
+          - fedora-rawhide-x86_64
```

Comment:
Fedora rawhide could be easily broken. It might be a better idea to use last stable version of fedora as the target here.

Reply:
Yeah, you're right.  Addressed in 541cb51.